library/contracts/safety/src/runtime.rs

```diff
@@ -0,0 +1,15 @@
+use proc_macro::TokenStream;
+
+/// For now, runtime requires is a no-op.
+///
+/// TODO: At runtime the `requires` should become an assert unsafe precondition.
+pub(crate) fn requires(_attr: TokenStream, item: TokenStream) -> TokenStream {
+    item
+}
+
+/// For now, runtime requires is a no-op.
+///
+/// TODO: At runtime the `ensures` should become an assert as well.
```

Comment:
I'd suggest to create a GitHub issue to track these TODOs.

Reply:
Sure... I'll create one

Reply:
https://github.com/model-checking/verify-rust-std/issues/17